src/analysis/type_utils.ml

```diff
@@ -78,7 +81,7 @@ module Printtyp = struct
               Btype.iter_type_expr (iter (pred d)) ty0
           end
       in
-      iter !verbosity ty;
+      iter verbosity ty;
```

Comment:
Lets be conservative since we did not do the "remove the reference" refactoring.
```suggestion
      iter !verbosity ty;
```